charts/rancher-istio/rancher-istio/1.10.200/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+annotations:
```

Comment:
This needs updated: https://github.com/rancher/rancher/issues/33528
It looks like 1.8.6 is in that list already but not 1.9.x ? 

Failing CI too.

Reply:
I will get the CI updated and update the 2.5.10

We included 1.9.x in 2.5.9 charts because it was OOB between 2.5.8 and 2.5.9 https://github.com/rancher/rancher/issues/33557

Reply:
I'm guessing that included the "version scheme change" but 1.8 didn't, so now there is a new 1.8 but not 1.9. Only 1.10 now would be airgapped.

Reply:
Yes, the OOB release for 1.9 did include the version changes, except for we did not bump the chart annotation for auto install. To make sure we are all up to date and all the new version schemes are on the same page, I did go ahead and bump the chart annotation in 1.9.6 from `catalog.cattle.io/auto-install: rancher-kiali-server-crd=1.32.100` to `catalog.cattle.io/auto-install: rancher-kiali-server-crd=1.32.100+up1.32.0`